test/distributed/_composable/test_replicate.py

```diff
@@ -136,6 +133,38 @@ def test_replicate_single_module(self):
         replicate_model = replicate(deepcopy(model))
         self._compare_module(model, replicate_model)
 
+    @skip_if_lt_x_gpu(2)
```

Comment:
default world size in `MultiProcessTestCase` is 4. We will need to override `world_size` property or change this to 4 as well.

```python
@property
    def world_size(self) -> int:
      return 2
```